src/assets/scripts/client/canvas/CanvasController.js

```diff
@@ -1526,12 +1526,21 @@ k
         cc.save();
         cc.translate(this.canvas.panX, this.canvas.panY);
 
+        // TODO: Remove the jQuery in favor of _each()!
         $.each(airport.terrain || [], (elevation, terrainLevel) => {
+            if (elevation < 1000) {
+                console.warn(`${airport.icao}.geojson contains 'terrain' at or` +
+                    ' below sea level, which is not supported!');
+
+                // within `$.each()`, this return acts like `continue;`
+                return;
+            }
+
             max_elevation = Math.max(max_elevation, elevation);
-            const color = `rgba(${UiController.terrain.COLOR[elevation]}, `;
+            const color = `hsla(${this.theme.TERRAIN.COLOR[elevation]}`;
```

Comment:
the idea was that since the terrain is many layers deep, hsla would make it easier to create a progressive lightness shift or adjust the overall perceived opacity while maintaining the same hue. That could be done in rgb, but would require back and forth with a web color thing to make the adjustments, while tweaking the h/s/l meant a more convenient way to get the terrain looking the way you wanted.